src/_nebari/stages/infrastructure/__init__.py

```diff
@@ -435,37 +439,42 @@ class AmazonWebServicesProvider(schema.Base):
             instance="m5.xlarge", min_nodes=1, max_nodes=5, single_subnet=False
         ),
     }
-    existing_subnet_ids: typing.List[str] = None
-    existing_security_group_ids: str = None
+    existing_subnet_ids: typing.Optional[typing.List[str]] = None
+    existing_security_group_ids: typing.Optional[str] = None
```

Comment:
I prefer "from typing import ...", but for consistency within the module, I'll leave it like this.